edi-x12-as2/camel/App.java

```diff
@@ -0,0 +1,4 @@
+//DEPS org.apache.activemq:artemis-jakarta-server:2.38.0
+//DEPS org.apache.camel:camel-http:4.10.0
+//DEPS org.smooks.cartridges.edi:smooks-edi-cartridge:2.1.0
+//DEPS org.smooks.cartridges:smooks-templating-cartridge:2.1.2
```

Comment:
comment: there is also `camel.jbang.dependencies` property in `application.properties` which can be used.